Dockerfile

```diff
@@ -18,6 +18,23 @@ RUN [ "$SKIP_TESTS" = 'false' ] && \
   bin/validate-values.sh && \
   bin/validate-templates.sh || true
 
+#-----------------------------
+FROM bats/bats:latest as test
```

Comment:
I am wondering if we should make a custom image with all dependencies
otomi/core-test where we run all tests including those from build stage.
It would significantly increased image build time and provide a consistent way of disabling all tests.
Moreover it would be more stable baseline, as we install latest dependencies that can change any time.
@Morriz what do you think?

Reply:
Yes, I can agree with that, you mean of course "it would significantly reduce image build time". However, I compressed the amount of layers of the commits anyway by having only one RUN and one COPY. The image is supposed to be extended per the Docker documentation.

Reply:
I don't think this makes sense seeing that the only test dep is bats. The rest is runtime deps.

I think we should add bats to tools and that's it.